DataManagement/GitTutorial.Rmd

```diff
@@ -236,11 +236,11 @@ However, if you are collaborating on a project we strongly suggest you to use fo
 <br/><br/>
 <br/><br/>
 
-## Gitignore file
+## .gitignore file
 
-When creating a new GitHub repository you can add a gitignore file.
-The gitignore defines which files can be commited and pushed to GitHub. We suggest to commit only code and plain text. 
-Do not commit data or output files. 
+When creating a new GitHub repository you can add a .gitignore file.
+The .gitignore defines which files can be commited and pushed to GitHub. We suggest to commit only code and plain text. 
+In general only data or output files are commited.
```

Comment:
In general,  data or output files are not committed. ?????

Reply:
ohhh too tired...